src/components/Dropdown/DropdownInput.tsx

```diff
@@ -37,15 +37,15 @@ export function DropdownInput(props: {
 
   const handleChange = useCallback((e: ChangeEvent<HTMLInputElement>) => {
     toggleDropdown(true);
+    onChange?.(e.target.value);
     updateFocusedItem(-1, e.target.value);
     setLastTypedOrSubmittedValue(e.target.value);
-    onChange?.(e.target.value);
   }, [onChange, setLastTypedOrSubmittedValue, toggleDropdown, updateFocusedItem]);
 
   const handleKeyDown = useCallback((e: KeyboardEvent<HTMLInputElement>) => {
     if (e.key === 'Enter' && (!submitCriteria || submitCriteria(focusedIndex))) {
+      updateFocusedItem(focusedIndex, typeof focusedItemData?.displayName === 'string' ? focusedItemData.displayName : undefined);
```

Comment:
how come we are using the display name of the focusedItemData and not the value field (which `updateFocusedItem` uses by default if the second arg here is not specified)?

Reply:
oop good catch

Reply:
I have to specify it here because now arrowing/tabbing doesn't change the value in the input by default